java/src/main/java/com/plaid/quickstart/QuickstartApplication.java

```diff
@@ -41,7 +42,7 @@ public String getName() {
   public void initialize(final Bootstrap<QuickstartConfiguration> bootstrap) {
     bootstrap.setConfigurationSourceProvider(
       new SubstitutingSourceProvider(bootstrap.getConfigurationSourceProvider(),
-                       new EnvironmentVariableSubstitutor()
+                       new EnvironmentVariableSubstitutor(false)
```

Comment:
This false config is required for the optional oauth parameters so that compiler does not complain.